ge/org.osate.ge/src/org/osate/ge/services/DiagramExportService.java

```diff
@@ -34,7 +34,7 @@
 import org.eclipse.core.resources.IFile;
 
 /**
- * Service for exporting a diagram as an image.
+ * Service for exporting a diagram as an image. Editor implements provide this service as a global service.
```

Comment:
"Editor implements provide this service" needs to be reworded

Reply:
Fixed.